Documentation/DataLoadEngine/MDFAttacher.md

```diff
@@ -0,0 +1,21 @@
+# MDF Attacher
+
+The MDF attacher is used for loading a detached database file into RAW.  
+This attacher does not load RAW tables normally (like AnySeparatorFileAttacher etc) instead it specifies that it is itself going to act as RAW.
+Using this component requires that the computer running the data load has file system access to the RAW SQL Server data directory (and that the path is the same).
```

Comment:
Path the same? Doesn't the override allow us to upload to \\sql-de-04\data\project4.mdf then load that as /db/data/project4.mdf ?

Reply:
Yes, I wasn't clear in the docs. It's assuming that your ldf and mdf file are in the same location (/var/data/project4.mdf and /var/data/project4_log.ldf). Will update